en_US.json

```diff
@@ -1921,6 +1921,7 @@
 	"report.vol.label.edge.traffic": "Edge Traffic",
 	"report.vol.label.origin.traffic": "Origin Traffic",
 	"report.vol.label.traffic.vol": "Traffic Volume",
+	"reportQuery.page.check.query.propertyHostnames": "query as property hostname",
```

Comment:
@jungho73  can you clarify how this text is used?  It sounds a bit strange

Reply:
@burtonquantil  Used as a checkbox title below the hostname edit box

Reply:
@jungho73  OK, I saw Bin's recommendation of this text.  I guess I'll try it when visible in our test environment and will comment if I think it should be reworded. Thanks.